celery_serverless/watchdog.py

```diff
@@ -23,6 +24,25 @@
 DEFAULT_STARTED_TIMEOUT = 30 # half minute
 
 
+def _get_watchdog_lock(lock_url='', lock_name='', default=dummy_threading.Lock, enforce=True) -> '(Lock, str)':
```

Comment:
Since `_get_watchdog_lock` and `_get_client_lock` should both work with redis locks, shouldn't their code be shared? It seems to me that there is a reasonable amount of copy/paste between both functions.

Reply:
Yes. Will be merged.

This is the 1st test-passing code version anyway ;) Lets get it better before merge.

Reply:
Nice! 👍

Reply:
Done. Take a look on `_get_lock` plz